pages/mesosphere/dcos/2.2/storage/external-storage/csi/index.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: layout.pug
+navigationTitle:  CSI Storage
+title: CSI Storage
+menuWeight: 20
+excerpt: Using CSI volumes with Marathon
+render: mustache
+model: /mesosphere/dcos/2.2/data.yml
+beta: false
+enterprise: false
+---
+
+DC/OS integrates with storage backends that use the Container Storage Interface (CSI) to interact with the cluster orchestrator (DC/OS in this case). This recently-developed standard is gaining wide adoption in the cloud-native ecosystem. If you want a storage solution which will be developed and supported long into the future, it may be wise to choose one with CSI support.
+
+Note that CSI volumes in DC/OS currently must be pre-provisioned before they are used by a service; dynamic provisioning of CSI volumes on demand is not yet supported. If you require dynamic provisioning, we currently recommend that you use [DVDI volumes][1] in DC/OS.
```

Comment:
The existing external volumes docs use this convention, which is why I've continued it. I'll change it to inline the links so that dead links are no longer possible.